examples/example_differential_evo_regression.py

```diff
@@ -126,7 +126,7 @@ def recording_callback(pop):
 
     history, champion = evolution()
 
-    print(f"Final expression {champion.to_sympy()[0]} with fitness {champion.fitness}")
+    print(f"Final expression {champion.to_sympy()[0]} " f"with fitness {champion.fitness}")
```

Comment:
why this change?

Reply:
hmm... i had to do a linebreak to make `flake` happy... and then I guess `black` removed it again?

Reply:
that's strange. could you try removing the `" f"` again and see what travis thinks? maybe after all you need to run flake with `flake8 --config=.flake8 .` in contrast to what i might've said :P